### PR TITLE
Fix build: re-running codegen

### PR DESCRIPTION
**Pull request by isoos**
Similar to the pub site, a dependency upgrade requires re-running.

### CODE REVIEW

**Comment by isoos** [comment]
strange: the generated code doesn't seem to be that much different, and looks like as if it wasn't able to deserialize an enum

File: lib/src/model.g.dart [hunk 1]
```diff
@@ -232,14 +229,12 @@ abstract class _$PenaltySerializerMixin {
...
-                (json['uses'] as Map).values.map((e) => e == null
-                    ? null
-                    : PlatformUse.values
-                        .singleWhere((x) => x.toString() == 'PlatformUse.$e'))),
+        (json['uses'] as Map<String, dynamic>)?.map((k, e) => new MapEntry(
+            k,
+            e == null
+                ? null
+                : PlatformUse.values
+                    .singleWhere((e) => e.toString() == 'PlatformUse.$e'))),
```
Comment (isoos) [inline lib/src/model.g.dart]: The issue **is** in the generated code: Here we have two `e` variables: one from the top-level `map`, one inside the `singleWhere`, and with that, `(e) => e.toString() == 'PlatformUse.$e')` is evaluated as false all the time.

@kevmoo: do we have a quick fix for this, or shall I downgrade the versions?

Comment (kevmoo) [reply lib/src/model.g.dart]: Gah! Downgrade. File an issue on json_serial – will fix ASAP

Comment (kevmoo) [reply lib/src/model.g.dart]: https://github.com/dart-lang/json_serializable/issues/180

Comment (kevmoo) [reply lib/src/model.g.dart]: https://github.com/dart-lang/json_serializable/pull/181

Comment (kevmoo) [reply lib/src/model.g.dart]: Publish @isoos !

Comment (isoos) [reply lib/src/model.g.dart]: Thanks, this works great!

**Comment by isoos** [comment]
PTAL

**Pull request merged**